app/models/percentage.rb

```diff
@@ -4,11 +4,11 @@ class Percentage < ActiveRecord::Base
   has_many :languages, :through => :percent_languages
 
   validates :country, presence: true
-  validates :percent, presence: true, :numericality => {:greater_than => 0, :less_than => 100}
+  validates :percent, presence: true, :numericality => {:greater_than => 0, :less_than_or_equal_to => 100}
```

Comment:
Options on new lines, and try to use Ruby 1.9+ syntax instead of hash rockets. For hashes it doesn't work sometimes in rare cases, so you might have to use has rockets there, test it out